static/js/publisher/form/media.test.js

```diff
@@ -0,0 +1,314 @@
+import Media from "./media";
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+
+describe("Media", () => {
+  const imageSelector = ".js-media-item-holder:not(.is-empty)";
+  const addImageSelector = ".p-listing-images__add-image";
+
+  it("should render without an image", () => {
+    const { container } = render(
+      <Media mediaData={[]} updateState={() => {}} />
```

Comment:
Is setting these props necessary? They seem to be set in `defaultProps` already to the same values. So I guess tests could only set the ones that are different(?)